src/frontend/handlers.go

```diff
@@ -102,17 +103,18 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 	plat.setPlatformDetails(strings.ToLower(env))
 
 	if err := templates.ExecuteTemplate(w, "home", map[string]interface{}{
-		"session_id":    sessionID(r),
-		"request_id":    r.Context().Value(ctxKeyRequestID{}),
-		"user_currency": currentCurrency(r),
-		"show_currency": true,
-		"currencies":    currencies,
-		"products":      ps,
-		"cart_size":     cartSize(cart),
-		"banner_color":  os.Getenv("BANNER_COLOR"), // illustrates canary deployments
-		"ad":            fe.chooseAd(r.Context(), []string{}, log),
-		"platform_css":  plat.css,
-		"platform_name": plat.provider,
+		"session_id":      sessionID(r),
+		"request_id":      r.Context().Value(ctxKeyRequestID{}),
+		"user_currency":   currentCurrency(r),
+		"show_currency":   true,
+		"currencies":      currencies,
+		"products":        ps,
+		"cart_size":       cartSize(cart),
+		"banner_color":    os.Getenv("BANNER_COLOR"), // illustrates canary deployments
+		"ad":              fe.chooseAd(r.Context(), []string{}, log),
+		"platform_css":    plat.css,
+		"platform_name":   plat.provider,
+		"is_cymbal_brand": isCymbalBrand,
```

Comment:
The only logic that changes here is the addition of `"is_cymbal_brand": isCymbalBrand,`.